cli/cycode.py

```diff
@@ -127,7 +133,26 @@ def get_cycode_client(client_id, client_secret):
         if not client_secret:
             raise click.ClickException("Cycode client secret is needed.")
 
-    return ScanClient(client_secret=client_secret, client_id=client_id)
+    if dev_mode:
```

Comment:
lets move the if and the scan client creation to scan_config, then import the scan_client and return it in line 141

Reply:
Fixed